packages/ingest/parse-pdr.js

```diff
@@ -40,6 +40,40 @@ function getItem(spec, pdrName, name, must = true) {
   return null;
 }
 
+/**
+ * Validate parsedSpec matches expectations
+ *
+ * @param {Object} spec - parsed spec
+ * @returns {Object} validated spec
+ * @throws {Error}
+ */
+function validateParsedSpec(spec) {
```

Comment:
Would `validateChecksumInformation` be a better name for what this function is doing?